packages/runtime-core/src/index.ts

```diff
@@ -79,7 +79,7 @@ export {
 
 // For getting a hold of the internal instance in setup() - useful for advanced
 // plugins
-export { getCurrentInstance } from './component'
+export { getCurrentInstance, getExposeProxy } from './component'
```

Comment:
I'm not sure that we should expose this from runtime-core, as this way it will be re-exposed from the main vue package. But really it's an internal API.

We could move to the `shared` package like other stuff that's internally shared between different renderers.

Reply:
What you said makes sense, but moving `getExposeProxy` to `shaerd` seems to be more costly, because the `publicPropertiesMap` it uses involves a lot of internal APIs.
Import `getExposeProxy` directly through the path, how do you think this is possible?
````
import { getExposeProxy } from '../../runtime-core/src/component'
````
Because I see that there are similar writing in [ `customFormatter.ts` ](https://github.com/vuejs/core/blob/main/packages/runtime-core/src/customFormatter.ts#L3) and [`packages/vue-compat/src/index.ts`](https://github.com/vuejs/core/blob/main/packages/vue-compat/src/index.ts#L7)